pkg/catalog/routes.go

```diff
@@ -642,10 +642,7 @@ func (mc *MeshCatalog) listPoliciesFromTrafficTargets(sa service.K8sServiceAccou
 }
 
 func isValidTrafficTarget(t *target.TrafficTarget) bool {
-	if t.Spec.Rules == nil || len(t.Spec.Rules) == 0 {
-		return false
-	}
-	return true
+	return t.Spec.Rules != nil && len(t.Spec.Rules) > 0
```

Comment:
May not be needed now, but this should also check `t != nil && t.Spec != nil` to avoid a nil pointer dereference.

And a nit: since `len()` evaluates to 0 for nil slices, the explicit nil check is redundant.

Reply:
good points - will make a PR in a few minutes